docs/detections/query-alert-indices.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[query-alert-indices]]
+== Query alert indices
+This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the [Alert schema].
+
+[discrete]
+=== Alert indices
+Alert events are stored in hidden {es} indices. The naming conventions for these indices and their aliases differ depending on which version of {stack} you're running:
+
+* *8.0.0 and later:* `.internal.alerts-security.alerts-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
+
+* *Before 8.0.0:* `.siem-signals-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
+
+[discrete]
+=== Alert index aliases
+We recommend querying the following index aliases to avoid querying the numbered alert indices:
+
+* If you're using version 8.0.0 or later: `.alerts-security.alerts-<space-id>`
+This alias includes both the legacy (before 8.0.0) and the new alert indices.
+
+* If you're using a version before 8.0.0: `.siem-signals-<space-id>`
```

Comment:
Consider either changing the first part to "8.0.0 and later:" and "Before 8.0.0:" to match the previous section, OR change the previous section to match the phrasing in this list. No strong preference either way, other than try to use parallel form for both lists.